tests/dataconverter/test_convert.py

```diff
@@ -17,15 +17,17 @@
 #
 """Test cases for the convert script used to access the DataConverter."""
 
```

Comment:
I do not see any reflection on the test. The test can also be modified if needed.

Reply:
I removed the `--input-file` inputs except for `test_cli` as we cannot test this like this. click is already checking that the file we pass is existing, therefore we never start parsing there. But I think it's fine to still test the option there.

Reply:
There are some `--input-file` options in the test file and they can be removed.

Reply:
No the only occurrence left is https://github.com/FAIRmat-NFDI/pynxtools/blob/f982c0e16636a6a5f87870f868014ab47188b871/tests/dataconverter/test_convert.py#L126-L127 which cannot be removed, because click automatically checks if `test_input` exists (and it doesn't). We can remove this test entirely but I would keep it until we completely remove `--input-file`.